.gitignore

```diff
@@ -50,3 +50,4 @@ yarn-debug.log*
 
 # Ignore Jetbrains IDE files
 /.idea
+/.vscode
```

Comment:
```suggestion
```

`/.vscode` isn't anything like Jetbrains IDE `/.idea` folders and contains project scoped configurations like `extensions.json`, `tasks.json`, `settings.json`, etc. that you want to share.

Reply:
I'll remove it for now